target_mysql/connector.py

```diff
@@ -196,13 +196,20 @@ def pick_individual_type(jsonschema_type: dict) -> sqlalchemy.types.TypeEngine:
         Returns:
             An instance of the appropriate SQL type class based on jsonschema_type.
         """
+        picked_type = None
         if "null" in jsonschema_type["type"]:
-            return None
-        if "integer" in jsonschema_type["type"]:
-            return BIGINT()
-        if jsonschema_type.get("format") == "date-time":
-            return TIMESTAMP()
-        return th.to_sql_type(jsonschema_type)
+            pass
+        elif (
+            "integer" in jsonschema_type["type"] or "decimal" in jsonschema_type["type"]
```

Comment:
Decimal is not a BIGINT, we should probably make a test just for Decimals. 

See https://dev.mysql.com/doc/refman/8.0/en/fixed-point-types.html

Reply:
Switched to using `DECIMAL(65,30)` for jsonschema "number" type.